src/equations/compressible_navier_stokes_2d.jl

```diff
@@ -153,12 +153,12 @@ function flux(u, gradients, orientation::Integer,
     # Components of viscous stress tensor
 
     # (4/3 * (v1)_x - 2/3 * (v2)_y)
-    tau_11 = 4.0 / 3.0 * dv1dx - 2.0 / 3.0 * dv2dy
+    tau_11 = 4 / 3 * dv1dx - 2 / 3 * dv2dy
     # ((v1)_y + (v2)_x)
     # stress tensor is symmetric
     tau_12 = dv1dy + dv2dx # = tau_21
     # (4/3 * (v2)_y - 2/3 * (v1)_x)
-    tau_22 = 4.0 / 3.0 * dv2dy - 2.0 / 3.0 * dv1dx
+    tau_22 = 4 / 3 * dv2dy - 2 / 3 * dv1dx
```

Comment:
These are cases that we should keep on a list of more involved changes (also in 3D) since `4 / 3` will be a `Float64` (but the current changes here are fine for now).